deploy/kubeturbo/templates/configmap.yaml

```diff
@@ -20,8 +20,12 @@ data:
       },
       {{- if .Values.featureGates }}
       "featureGates": {
-        "disabledFeatures": [{{ .Values.featureGates.disabledFeatures }}]
-      }
+        {{$first := true}}
+        {{- range $key, $val := .Values.featureGates }}
+        {{- if $first}}{{$first = false}}{{else}},{{-end}}
+        {{ $key | quote }}: {{ $val }}
+        {{- end }}
+      },
```

Comment:
Have we considered something like:
`{{  .Values.featureGates | toJson }}` ?

See: [toJson](https://helm.sh/docs/chart_template_guide/function_list/#tojson-musttojson)

Reply:
Changed to this. Thanks for the recommendation.